lib/runner/request-helpers-postsend.js

```diff
@@ -16,6 +16,7 @@ module.exports = [
         if (!(context.auth && context.auth.type)) { return done(); }
 
         var auth = context.auth,
+            originalAuthParams = context.originalItem.getAuth().parameters(),
```

Comment:
Make sure that `originalItem` has an `auth`. Eventhough, it will always be if copied from `item`, but we don't want to rely on the assumption.